src/models/nebula36-1/assembly.md

```diff
@@ -0,0 +1,399 @@
+- [Installing the CPU shroud fans](#installing-the-cpu-shroud-fans)
+- [Installing the GPU](#installing-the-gpu)
+- [Installing the SATA backplane](#installing-the-sata-backplane)
+- [Installing 2.5" drives](#installing-25-drives)
+- [Replacing the bottom case fan](#replacing-the-bottom-case-fan)
+- [Installing side intake fans](#installing-side-intake-fans)
+- [Installing the power supply](#installing-the-power-supply)
+- [Wiring the front I/O and power button](#wiring-the-front-io-and-power-button)
+
+## Replacing the front accent strip:
+
+nebula36 includes a customizable accent panel on the front of the case, which can be swapped to change the case's look and feel. The instructions for swapping the accent are also available in [video form](https://system76.com/r/diyaccentpanel).
+
+- **Tools required:** None  
+
+### Steps to replace the front accent strip:
+
+1. Place the nebula36 on the edge of the desk so the front side is hanging off of the desk.
+    - The nebula36 can alternatively be lifted or tilted so the front of the computer is hovering above the desk.
+2. Slide the accent strip down to unlock it.
+    - The accent can be gripped at the bottom edge.
+3. Pull the accent strip off of the case, starting with the bottom edge.
+4. Place the new accent strip onto the front of the case and slide it up to lock it into place.
+
+![Front accent strip removal](./img/accent-strip-removal.webp)
+
+## Removing the top case:
+
+The top case can be removed to access the internal components.
+
+- **Tools required:** Cross-head (Phillips) screwdriver (optional)  
+
+### Steps to remove the top case:
+
+1. Remove the four outer screws holding the top case onto the machine.
+
+![Top case screws](./img/top-case-screws.webp)
+
+2. Slide the top case up and off of the machine.
+
+## Removing the GPU brace:
+
+The GPU brace provides a mounting point to help keep installed GPUs (or other PCI Express cards) in place while transporting the system. It can also be used to mount up to two extra intake fans (one, a `BQ SIW4-12025-MF-PWM`, is optional when purchasing the chassis; a second is not included.)
+
+![GPU brace](./img/gpu-brace.webp)
+
+The fan splitter on the GPU brace is intended for fans mounted on the GPU brace, and should not be used for the CPU shroud fans or the bottom intake fan.
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to remove the GPU brace:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case).
+2. Unscrew the four screws holding the GPU brace in place (two on the front of the case, two on the back.)
+
+![GPU brace screws](./img/gpu-brace-screws.webp)
+
+3. Pull the GPU brace out of the chassis.
+
+## Removing the CPU shroud:
+
+The CPU shroud supports one or more of the CPU fans and guides airflow from the side of the case through the CPU radiator to the back exhaust vent. The CPU shroud is custom-designed to work with the Noctua `NH-U12S` CPU cooler that's included with nebula36.
```

Comment:
>the Noctua `NH-U12S` CPU cooler that's included with nebula36.

Should be "that's an optional add-on with nebula36" if the cooler isn't a standard inclusion

Reply:
Addressed.